### PR TITLE
DOCS-10089 Update manage-backup-restore.adoc

### PR DESCRIPTION
**Pull request by akrag**
As part rtfctl back up and restore feature, We want to include the following in the public documentation, 

- Current documentation should include common procedure for backup and restore for both BYOK and  Appliance Clusters. 
- Documentation common procedure needs to include a section for scaling down/stopping the original cluster before restoring it to a new cluster.
- Separate out notes for changes/assumptions for backup and restore on BYOK and Appliance Clusters

### CODE REVIEW

**Review by sleslie23 — DISMISSED** [review]

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 1]
```diff
@@ -1,63 +1,93 @@
...
+====
+Configuration changes to deployed applications and management services made after a back up cannot not be restored.  
+====
 
 [NOTE]
-The state of Runtime Fabric is distributed across its controller nodes. Use the following procedures to back up and restore
-the required state necessary for all deployed applications, management services, and their configurations. 
-Individual node backup and restore operations are not supported.
+====
+For Runtime Fabric running on VMs / Bare Metal, the backup process also includes backing up of underneath Kubernetes appliance related custom resources [logforwarder, smtp, alert and alerttarget].
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/manage-backup-restore.adoc]: For clarity, maybe "...also includes backing up the custom resources underneath the Kubernetes appliance"? 
I can't quite figure out what's underneath what and what's related to what...

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 2]
```diff
@@ -1,63 +1,93 @@
...
 ----
+
 This command creates an archive of the current system state in `<path_to_backup_file>`, which can be any path in the file system that you have write access to, for instance `/opt/anypoint/runtimefabric/backup.tar.gz`. 
 
-. Move the archive file to secure storage (a different machine that is not part of the RTF cluster);
-for example:
-+
+[NOTE]
+====
+For Runtime Fabric running on VMs / Bare Metal, The `rtfctl` binary is present in directory `/opt/anypoint/runtimefabric/`. Confirm that you run the backup command on any controller node in the cluster as a privileged user. To use the archive for restoring later, Copy the archive file to secure storage that is on different machine and is not part of the RTF cluster. The following command, for example, uses `scp` to copy the archive to another drive:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/manage-backup-restore.adoc]: lower case "the" after "Bare Metal,"

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/manage-backup-restore.adoc]: also, lower case "Copy" in "copy the archive file"
"on a different machine" (missing article 'a')

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 3]
```diff
@@ -1,63 +1,93 @@
...
+[NOTE]
+====
+To create a new Kubernetes cluster on VMs / Bare Metal without Runtime Fabric being installed, run the Runtime Fabric installer script(s) but do not provide the fabric activation data.
+====
 
 [WARNING]
-You can not restore Runtime Fabric on a newer version of the Runtime Fabric cluster. The new installation must use 
-the same version of Runtime Fabric installation bundle as the backup.
+====
+For restoring on existing Runtime Fabric cluster, You must restore on to the same version of Runtime Fabric that you used to create the backup archive. You can not restore from an older version to a newer version of Runtime Fabric running on the cluster.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/manage-backup-restore.adoc]: lower case for "you"

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/manage-backup-restore.adoc]: "cannot" not "can not"

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 4]
```diff
@@ -1,63 +1,93 @@
...
+====
 
-. Log in to a controller node in your Runtime Fabric as a privileged user (with root access).
+. Copy backup archive file and make sure that it is available to `rtfctl`.
+
+[NOTE]
+====
+For Runtime Fabric on VMs / Bare Metal, copy the compressed backup archive file to a folder in any of the controller nodes of the environment to be restored. For example You can transfer this file securely via the following command:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/manage-backup-restore.adoc]: lower case for "you"

**Review by Mulesoft-Mysti — DISMISSED** [review]
a few suggested tweaks

File: modules/ROOT/pages/manage-backup-restore.adoc (hunk 1 above)
Comment (akrag) [reply modules/ROOT/pages/manage-backup-restore.adoc]: Runtime Fabric runs on top of Kubernetes Appliance (called Gravity)  and we backing up custom resources done on the Kubernetes appliance. I am not sure if we can mention our Vendor Gravity in the documentation.

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 5]
```diff
@@ -1,63 +1,93 @@
 = Back up and Restore Runtime Fabric
 
-To preserve applications in the event of system failure, ensure that Runtime Fabric is automatically backed up. 
-MuleSoft highly recommends configuring hourly backups. Store your backup archive on external storage outside 
-the Runtime Fabric cluster.
+To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup archive on external storage outside the Runtime Fabric cluster.
+
+The state of Runtime Fabric is distributed across the Kubernetes cluster. Use the following procedures to back up and restore the required state necessary for all deployed applications, management services, and their configurations. Individual node backup and restore operations are not supported.
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/manage-backup-restore.adoc]: `Individual node backup and restore operations are not supported` we can perhaps clarify this better? Perhaps VM snapshot backup/restore is not supported?

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 6]
```diff
@@ -1,63 +1,93 @@
...
 ----
 
+For Runtime Fabric running on Self-Managed Kubernetes, confirm that the `rtfctl` binary is present in the current directory and Kubernetes(kubectl) context is set to the intended backup cluster.
+====
+
 == Restore a Cluster
 
-To restore a Runtime Fabric cluster from a backup, you can use the existing Runtime Fabric cluster or create a new Runtime Fabric 
-cluster with the same hardware (number of servers, disks, and so on) as the cluster being restored.
+To restore Runtime Fabric cluster from a backup, use an existing Runtime Fabric cluster or create a new Kubernetes cluster with the same configuration (number of servers, disks, and so on) as the backed up cluster.
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/manage-backup-restore.adoc]: `same configuration (number of servers, disks, and so on) as the backed up cluster. `
is this a hard requirement?

Comment (kmendiratta-crm) [reply modules/ROOT/pages/manage-backup-restore.adoc]: perhaps we should also clarify if the cluster being restored to is different, the clutser should be provisioned without an association to a new fabric (i.e. just a gravity cluster without runtime fabric software installed)?

Comment (akrag) [reply modules/ROOT/pages/manage-backup-restore.adoc]: @kmendiratta-crm 
- Same configuration is a hard requirement on the gravitational cluster but it should work for Self-managed Kubernetes. But I think this line will provide as a general guideline for backup and restore funcionality.
- This already provided in note section below

**Review by kmendiratta-crm — APPROVED** [review]

**Review by kmendiratta-crm — DISMISSED** [review]

File: modules/ROOT/pages/manage-backup-restore.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/manage-backup-restore.adoc]: No, we are not supposed to mention Gravity in the PCE/RTF docs. The only place it is exposed in the docs is in the name of one of the command line utilities (gravity).

File: modules/ROOT/pages/manage-backup-restore.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/manage-backup-restore.adoc]: In this context "Runtime Fabric running on VMs / Bare Metal" is the official product name. It should not be lower case.

Comment (sleslie23) [reply modules/ROOT/pages/manage-backup-restore.adoc]: RTF should be Runtime Fabric.

Comment (sleslie23) [reply modules/ROOT/pages/manage-backup-restore.adoc]: Runtime Fabric running on VMs / Bare Metal must be changed to `Runtime Fabric on VMs / Bare Metal` which is the official product name.

**Review by sleslie23 — DISMISSED** [review]

**Review by sleslie23 — DISMISSED** [review]

**Review by sarjeet2013 — DISMISSED** [review]
LGTM

**Review by sundippatel — APPROVED** [review]
Committed changes to improve readability. 

Approving with those changes in play (or with further improvements on top of those changes).

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 7]
```diff
@@ -2,15 +2,15 @@
 
 To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup archive on external storage outside the Runtime Fabric cluster.
 
-The state of Runtime Fabric is distributed across the Kubernetes cluster. Use the following procedures to back up and restore the required state necessary for all deployed applications, management services, and their configurations. Individual Kubernetes node VM snapshot backup/restore is currently not supported. 
+The state of Runtime Fabric is distributed across the Kubernetes cluster. Use the following procedures to back up and restore the required state necessary for all deployed applications, management services, and their configurations. Performing a backup per node (such as via a VM snapshot) is not supported.
 
 [WARNING]
 ====
-Configuration changes to deployed applications and management services made after a back up cannot not be restored.  
+Configuration changes to deployed applications and management services made after a back up will not be restored.
```
Comment (sleslie23) [inline modules/ROOT/pages/manage-backup-restore.adoc:9]: Our style guide prohibits using future tense.

**Review by sleslie23 — APPROVED** [review]

**Pull request merged**